sote/game/options.lua

```diff
@@ -1,13 +1,28 @@
 
-
-
 local opt = {}
 
+---@alias Fullscreen "false" | "exclusive" | "desktop"
+---@class Options
+---@feild ["volume"] number
+---@feild ["fullscreen"] Fullscreen
+---@feild ["fitscreen"] boolean
+---@feild ["rotation"] boolean
+---@feild ["update_map"] boolean
+---@feild ["treasury_ledger"] number
+---@feild ["debug_mode"] boolean
+---@feild ["zoom_sensitivity"] number
+---@feild ["camera_sensitivity"] number
+---@feild ["exploration"] number
+---@feild ["travel-start"] number
+---@feild ["travel-end"] number
+---@feild ["screen_resolution"] {width number, height number}
 
+---@return Options
 function opt.init()
 	return {
 		["volume"] = 0,
-		["fullscreen"] = true,
+		["fullscreen"] = "false",
+		["fitscreen"] = true,
```

Comment:
While this works, bitser (used for serialization in sote), uses love's built-in filesystem to serialize/deserialize objects.
Currently, if someone has a file with the previous version of the program, bitser will load the previous versions 

We ought to check for a version and if it's missing, reinitialize the options object.
Something like the following:
```lua
function opt.init()
    return {
        ["version"] = "v0.3.1"
        -- etc
```

Then, it'd need to be checked in the verify method:
```lua
function opt.verify()
	local default = opt.init()
        if OPTIONS.version ~= default.version then
            OPTIONS = default
            return
        end

	for i, j in pairs(default) do
		if OPTIONS[i] == nil then
			OPTIONS[i] = j
		end
	end
end
```
Something along those lines will ensure that users with older versions of the options schema will still be able to boot the game